.gitignore

```diff
@@ -6,3 +6,4 @@ benchmarks/leaks/input
 *.tgz
 .npmrc
 webpack
+out
```

Comment:
What is the purpose of this folder? I noticed we output ts build there, but why?

Reply:
`tsc` builds both common and browser as separate packages in the out dir. We need to do some post-processing before building a package (see https://github.com/ClickHouse/clickhouse-js/blob/32e4f51ebd13aa3de4a71520cdbb6dc19182418e/.scripts/build.sh)